evalml/tests/automl_tests/test_automl_search_classification_iterative.py

```diff
@@ -0,0 +1,421 @@
+from unittest.mock import MagicMock
+
+import pandas as pd
+import pytest
+
+from evalml import AutoMLSearch
+from evalml.automl.callbacks import raise_error_callback
+from evalml.automl.utils import get_best_sampler_for_data
+from evalml.model_family import ModelFamily
+from evalml.pipelines import (
+    BinaryClassificationPipeline,
+    MulticlassClassificationPipeline,
+)
+from evalml.pipelines.components.utils import get_estimators
+from evalml.pipelines.utils import make_pipeline
+from evalml.problem_types import ProblemTypes
+
+
+@pytest.mark.parametrize("automl_type", [ProblemTypes.BINARY, ProblemTypes.MULTICLASS])
```

Comment:
I believe every test in this file uses `allowed_component_graphs` so tagging as iterative.